packages/amplify-ui-components/src/components/amplify-confirm-sign-up/amplify-confirm-sign-up.tsx

```diff
@@ -214,6 +215,8 @@ export class AmplifyConfirmSignUp {
         break;
     }
     try {
+      if (!this.userInput) throw new Error('Username can not be empty');
```

Comment:
Can we use the following? 
```suggestion
      if (!this.userInput) throw new Error(Translations.EMPTY_USERNAME);
```

Reply:
Damn, can't believe I missed that one!